test/test_evaluation/test_train_evaluator.py

```diff
@@ -546,12 +546,13 @@ def test_file_output(self, loss_mock):
         configuration = unittest.mock.Mock(spec=Configuration)
         queue_ = multiprocessing.Queue()
         loss_mock.return_value = None
+        scorer_list = SCORER_LIST
 
         evaluator = TrainEvaluator(self.backend_mock, queue=queue_,
                                    configuration=configuration,
                                    resampling_strategy='cv',
                                    resampling_strategy_args={'folds': 5},
-                                   all_scoring_functions=True,
+                                   scoring_functions=scorer_list,
```

Comment:
What's the reason to assign the constant to a local variable and use the local variable here?

Reply:
Missed it. Made changes.